src/js/components/homepage/features/PaneFeature.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * PaneFeature.jsx
+ * Created by David Trinh 4/19/18
+ */
+
+import React from 'react';
+
+const PaneFeature = () => (
+    <div className="feature-pane">
+        <div className="feature-pane__wrapper">
+            <h2 className="feature-pane__title"><span>FEATURED CONTENT</span></h2>
+            <div className="feature-pane__content-wrapper">
+                <div className="feature-pane__content">
+                    <h2 className="feature-pane__content-title">Discover the Data Lab</h2>
```

Comment:
Since these titles are children of Feature Content, they should be `h3` tags to indicate that hierarchy.

Reply:
How about using the pseudo-elements after and before, instead of div?